libs/@guardian/atoms-rendering/src/YoutubeAtomOverlay.tsx

```diff
@@ -81,17 +84,46 @@ const playButtonStyling = css`
 	}
 `;
 
-const videoDurationStyles = css`
+const pillStyles = css`
 	position: absolute;
 	top: ${space[2]}px;
 	right: ${space[2]}px;
 	${textSans.xxsmall({ fontWeight: 'bold' })};
 	background-color: rgba(0, 0, 0, 0.7);
 	color: ${palette.neutral[100]};
-	padding: ${space[1]}px ${space[3]}px;
 	border-radius: ${space[3]}px;
+	padding: 0 6px;
+	display: inline-flex;
+`;
+
+const pillItemStyles = css`
+	/* Target all but the first element, and add a border */
+	:nth-of-type(n + 2) {
+		border-left: 1px solid rgba(255, 255, 255, 0.5);
+	}
+`;
+
+const pillTextStyles = css`
+	line-height: ${space[4]}px;
+	padding: ${space[1]}px 6px;
 `;
 
+const liveStyles = css`
+	::before {
+		content: '';
+		width: 9px;
+		height: 9px;
+		border-radius: 50%;
+		background-color: ${palette.news[500]};
+		display: inline-block;
+		position: relative;
+		margin-right: 0.1875rem;
+	}
+`;
+
+const capitalise = (str: string): string =>
+	str.charAt(0).toUpperCase() + str.slice(1);
```

Comment:
If you want to narrow it down even further 😉 

```suggestion
const capitalise = <T extends string>(str: T): Capitalize<T> =>
	str.charAt(0).toUpperCase() + str.slice(1);
```

Reply:
👆 the main problem with this approach is that you end up having to cast the type, which feels like you’re not gaining much.